Vienna/Sources/Main window/WebKitArticleTab.swift

```diff
@@ -107,6 +108,23 @@ class WebKitArticleTab: BrowserTab, ArticleContentView, CustomWKUIDelegate {
         // TODO: ignored intentionally. Find more elegant solution
     }
 
+    func updateLoadingProgress(_ oldProgress: Double) {
+        guard let animationLayer = self.view.layer else {
+            return
+        }
+
+        let toValue = max(0.75, Float(loadingProgress))
```

Comment:
I think for having a proper animation, this should be min, where the value is the same as max of fromValue

Reply:
Or something entirely different, like a spinner or a progress circle